src/components/Navbar.js

```diff
@@ -1,12 +1,141 @@
-import React from 'react'
-import styled from "styled-components"
+import React, { useState } from "react";
+import styled from "styled-components";
+import logo from "../images/logo.svg";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBars, faX } from "@fortawesome/free-solid-svg-icons";
+import { menuItems } from "../misc/menuItems";
+import { pallete } from "../misc/pallete";
+import Button from "./Button";
 
 export default function Navbar() {
-  return (
-    <header>
-        <nav>
+  const [isActive, setIsActive] = useState(false);
 
-        </nav>
-    </header>
-  )
+  return (
+    <Header>
+      <LogoContainer width="200px">
+        <img src={logo} alt="logo" />
+      </LogoContainer>
+      <NavbarContainer open={isActive}>
+        <Menu>
+          {menuItems.map((item) => {
+            return (
+              <li key={item.name}>
+                <a href={item.href}>{item.name}</a>
+              </li>
+            );
+          })}
+        </Menu>
+        <AuthOptions>
+          <Login>Login</Login>
+          <Button size={"small"} text={"Register"}></Button>
```

Comment:
jeśli prop jest stringiem to możesz zapisać to prościej -> size="small" (bez nawiasów klamrowych)

Reply:
i jeśli skorzystasz z tego co pisałem w komentarzu odnosnie obiektu z właściwościami - tutaj wtedy użyłbyś takiej notacji:

`size={buttonSize.small}`